static/js/components/dashboard/FinancialAidCard.js

```diff
@@ -0,0 +1,185 @@
+            omitDay={false}
+            validator={() => null}
+          />
+          <Button className="dashboard-button">
+            Submit
+          </Button>
+        </div>
+      </div>;
+    default:
+      // should not get here
+      return null;
+    }
+  }
+
+  inner() {
+    const {
+      program,
+      coursePrice,
+      openFinancialAidCalculator,
+    } = this.props;
+    if (!program.financial_aid_availability) {
+      return null;
+    }
+
+    const {
+      has_user_applied: hasUserApplied,
+      min_possible_cost: minPossibleCost,
+      max_possible_cost: maxPossibleCost,
+      application_status: applicationStatus,
+    } = program.financial_aid_user_info;
+    if (!hasUserApplied) {
+      return <div className="personalized-pricing">
+        <div className="heading">
+          How much does it cost?
+          { courseListToolTip('filler-text', 'course-price') }
+        </div>
+        <div className="explanation">
+          Courses cost varies between {price(minPossibleCost)} and {price(maxPossibleCost)} (full
+          price), depending on your income and ability to pay.
+        </div>
+        <button
+          className="mm-button dashboard-button"
+          onClick={openFinancialAidCalculator}
+        >
+          Calculate your cost
+        </button>
+      </div>;
+    }
+
+    switch (applicationStatus) {
+    case FA_STATUS_APPROVED:
+    case FA_STATUS_AUTO_APPROVED:
+    case FA_STATUS_REJECTED:
+    case FA_STATUS_SKIPPED:
+      return <Grid className="financial-aid-box">
+        <Cell col={12}>
+          Your cost is <b>{price(coursePrice.course_price)} per course</b>.
+        </Cell>
+      </Grid>;
+    case FA_STATUS_PENDING_MANUAL_APPROVAL:
+    case FA_STATUS_DOCS_SENT:
+    case FA_STATUS_PENDING_DOCS:
+      return <div>
+        <Grid>
+          <Cell col={12}>
+            Your cost is {price(coursePrice.course_price)} per course.
+          </Cell>
+        </Grid>
+
+        <Grid className="financial-aid-box">
+          <Cell col={12}>
+            Before you can pay, you need to verify your income by mailing or faxing one
+            of the following documents:
+            <ul>
+              <li>A notarized document verifying income</li>
+            </ul>
+          </Cell>
+        </Grid>
+
+        <Grid>
+          <Cell col={1} />
+          <Cell col={5}>
+            Mail to
+          </Cell>
+          <Cell col={6}>
+            or fax
+          </Cell>
+        </Grid>
+
+        <Grid>
+          <Cell col={1} />
+          <Cell col={5}>
+            MIT, Economics Department<br />
+            100 Main Street<br />
+            Cambridge, MA 02139
+          </Cell>
+          <Cell col={6}>
+            001 (999) 999-9999
+          </Cell>
+        </Grid>
+
+        <hr />
+        {this.documentStatus()}
+      </div>;
+    // FA_STATUS_CREATED should not be seen here
+    default:
+      return null;
+    }
+  }
+
+  render() {
```

Comment:
following the code path is a little difficult in this file. what do you think about refactoring `render` to something like this, and shifting the code around to support those new methods?

```javascript
render() {
  let body = '';
  if (program.financial_aid_availability) {
    if (!hasUserApplied) {
      body = this.renderInitialAidPrompt();
    } else {
      body = this.renderAidApplicationStatus();
    }
  }
  
  return <Card shadow={0}>
    <CardTitle>Pricing Based on Income</CardTitle>
      <div>
        { body }
      </div>
  </Card>;
}
```

i would also recommend we change `documentStatus` to `renderDocumentStatus`.

side note: this component absolutely should not be rendered if `program.financial_aid_availability` is false, correct? i don't think the logic to render nothing in that case should be a part of this file.